test/common/ssl/ssl_socket_test.cc

```diff
@@ -142,6 +142,7 @@ const std::string testUtilV2(const envoy::api::v2::Listener& server_proto,
                              const std::string& expected_client_cert_uri,
                              const std::string& expected_alpn_protocol,
                              const std::string& expected_stats, unsigned expected_stats_value,
+                             const std::string& expected_requested_server_name,
```

Comment:
Nit, but could you move this before `expected_alpn_protocol`?

Reply:
@PiotrSikora Sure, doing it now.